packages/babel-preset-react-app/create.js

```diff
@@ -145,6 +145,10 @@ module.exports = function(api, opts, env) {
         require('@babel/plugin-proposal-decorators').default,
         false,
       ],
+      // Adds Numeric Seperators for TypeScript files
+      isTypeScriptEnabled && [
```

Comment:
Not sure that we need to restrict this to TypeScript only. It seems to be a stable feature that has already shipped in Chrome.